src/ucp/core/ucp_mm.c

```diff
@@ -150,9 +151,8 @@ static ucs_status_t ucp_mem_alloc(ucp_context_h context, size_t length,
             }
         }
 
-        /* TODO: pass addr */
-        status = uct_mem_alloc(NULL, length, uct_flags, &method, 1, mds, num_mds,
-                               name, &mem);
+        status = uct_mem_alloc(memh->address, length, uct_flags, &method, 1, mds,
```

Comment:
okay...how we notify user about this ? We don't want user invoke the query every single time.